orc8r/tools/packer/scripts/cleanup.sh

```diff
@@ -13,19 +13,13 @@
 ################################################################################
 
 # Uninstall Ansible and remove PPA.
-apt -y remove --purge ansible
```

Comment:
I think the previous version works fine. Also, I think that we don't use this file for the `magma_dev` and `magma_test` VMs, so the changes to this file are not tested. Thus I would prefer not touching this file.